src/pages/index.tsx

```diff
@@ -56,7 +57,9 @@ export default function Index(): JSX.Element {
         <Hero title={title} subTitle={subTitle} />
 
         <section className="node-demo-container">
-          <div className="node-demo"></div>
+          <div className="node-demo">
```

Comment:
Think we can bake this wrapper into the `<DemoTabs>` component?

Reply:
Seems fair… I also wanted to separate the styles out with an `import` right into `DemoTabs.tsx`.

I'll do that for the next commit.